.speakeasy/workflow.lock

```diff
@@ -0,0 +1,14 @@
+speakeasyVersion: 1.277.4
+sources: {}
+targets: {}
+workflow:
+    workflowVersion: 1.0.0
+    speakeasyVersion: latest
+    sources:
+        livepeer-studio-api:
+            inputs:
+                - location: https://raw.githubusercontent.com/livepeer/studio/sk/schema-fixes/packages/api/src/schema/api-schema.yaml
```

Comment:
This is temporary and should be updated once the pr on the Studio repo has been merged